test/Shared/Shared.Tests.csproj

```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <RootNamespace>Microsoft.Shared.Test</RootNamespace>
     <Description>Unit tests for Microsoft.Shared</Description>
+    <DefineConstants>$(DefineConstants);TESTS_JSON_SCHEMA_EXPORTER_POLYFILL</DefineConstants>
```

Comment:
I hadn't previously comprehended that the JsonSchemaExporter tests were in Shared. That feels a bit off the mark for what I've perceived Shared to be, which is really about repo-wide supporting functionality. Should this all be moved to one of the specific M.E.AI test suites and then, if necessary to be used from a different one, just reference the code via relative path?

Reply:
At the end of the day it is a polyfill and we're using `Shared.Tests` to test polyfills. The other issue is that all our M.E.AI test projects run on STJv9 (forced by the main M.E.AI dependencies) so injecting the polyfill there results in conflict.